namelists

```diff
@@ -0,0 +1 @@
+namelists_24.07.02
```

Comment:
Should we have namelist symlinks on the root folder?

Reply:
I'm not sure what you mean.

Reply:
namelists is a symlink to namelists_24.07.02/ in case that's not obvious from the git diff view.

Reply:
Yup makes sense, I just thought `namelists_24.07.02/` and  `namelists_25.05.27` could have been not in the home folder, but something like `namelists/v_24.07.02` and `namelists/v_25.05.27` (the reasoning being that the home folder isn't populated.

Reply:
I've updated to put the version folders under namelists/. I want the CABLE configurations to get out of the benchcab configuration example, so I think we can leave it like this for now.